stix_shifter_modules/onelogin/stix_translation/json/operators.json

```diff
@@ -1,6 +1,7 @@
 {
     "ComparisonExpressionOperators.And": "&",
     "ComparisonComparators.Equal": "=",
+    "ComparisonComparators.In": "or",
```

Comment:
I don't think this is going to work. The native Onelogin query doesn't support OR. The work around for supporting the IN operator depends on the connector already having defined `ComparisonExpressionOperators.Or` in this object, which it doesn't.

Reply:
So we don't do the IN operator anymore?